tests/integration/customizations/s3/test_plugin.py

```diff
@@ -307,8 +307,9 @@ def test_mv_with_large_file(self):
                          len(file_contents.getvalue()))
 
     def test_mv_to_nonexistent_bucket(self):
+        bucket_name = random_bucket_name()
```

Comment:
Should this be static for the entire test runner? Otherwise, we're increasing probability of a collision. We should be able to come up with one "non-existant" name (maybe verify it?) and then run all the tests.

Reply:
In the latest commit I created a `_NON_EXISTENT_BUCKET` constant which is verified in the `setup_module` function where we verify other buckets.